style.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  margin: 0 auto;
+  height: 48px;
+  padding: 24px;
+}
+
+header .toolbar {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  color: #636fd4;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+p {
+  font-size: 4.3vw;
+}
+
+.my_logo {
+  margin-right: 200px;
+  margin-left: 10px;
+}
+
+#Introduction {
+  width: 100%;
+  height: calc(100vh - 48px);
+  background-image: url("images/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding-top: 114px;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+#Introduction h1 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  color: #253652;
+  padding: 0 24px;
+}
+
+#toppart {
+  display: flex;
+  justify-content: space-between;
+}
+
+#toppart p {
+  font-style: normal;
+  font-weight: 400px;
+  font-size: 16px;
+  color: #344563;
+  padding: 0 24px;
+}
+
+#introduction > a {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
+  text-decoration: none;
+  padding: 0 24px;
+}
+
+.Socials {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+  padding: 0 24px;
+}
+
+.btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #636fd4;
+  border-radius: 8px;
+  padding: 12px;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.btn:hover {
+  color: white;
+  background-color: #636fd4;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.btn:active {
+  background-color: rgb(31, 31, 241);
+}
+
+.btn:disabled {
+  color: #556175;
+  border: 1px solid #c1c7d0;
+}
+
+.mid {
+  padding: 20px;
```

Comment:
- [ ] Kindly consider removing the bottom padding from this element to lower the gap between this element and the element below it. you can do it by replacing this line with `padding: 20px 20px 0;` 
this will make our design look similar to the original one.